agent_ops.go

```diff
@@ -137,3 +139,15 @@ func (agent *Agent) FlushBucket(ctx context.Context, opts *cbmgmtx.FlushBucketOp
 func (agent *Agent) DeleteBucket(ctx context.Context, opts *cbmgmtx.DeleteBucketOptions) error {
 	return agent.mgmt.DeleteBucket(ctx, opts)
 }
+
+func (agent *Agent) Search(ctx context.Context, opts *cbsearchx.QueryOptions) (cbsearchx.QueryResultStream, error) {
+	return agent.search.Query(ctx, opts)
+}
+
+func (agent *Agent) UpsertSearchIndex(ctx context.Context, opts *cbsearchx.UpsertIndexOptions) error {
```

Comment:
UpsertIndex?

Reply:
This is on the agent. UpsertIndex would be quite ambiguous - at the very least that exists for both search and query.